docs/components/beacon/index.html

```diff
@@ -0,0 +1,77 @@
+---
+title: Beacon
+minver: 0.22.0
+also:
+  elements/hx-beacon: <hx-beacon>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    A beacon is an animation that attracts a user's attention
+    to a new feature or advanced functionality in a control panel.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="single-beacon">Beacon</h2>
+      <p>
+        The custom <code>{{page.title}}</code> element provides an animation
+        that attracts a user's attention to a new feature
+        or advanced functionality in a control panel. A beacon disappears after a user clicks it.
+      </p>
+    </header>
+
+    <div class="example" id="vue-beaconDemo" v-cloak>
+      <div>
+        <hx-beacon
+          @dismiss="onEvent"
+        >
+        </hx-beacon>
+      </div>
+
+      <footer>
+        <pre><code v-text="snippet"></code></pre>
+     </footer>
+    </div>
+  </section>
+
+
+  <section>
+    <header>
+      <h2 id="basic-menu">Working Example</h2>
+      {# TODO: add section description #}
+    </header>
+
+    <div class="example">
+      <div>
+        <hx-disclosure class="hxBtn" aria-controls="basicMenuId">
+          <hx-icon type="cog"></hx-icon>
+          <span>Actions</span>
+          <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+          <hx-beacon @dismiss="onEvent"></hx-beacon>
```

Comment:
This should not adjust the `<hx-menu>` size.

<img width="664" alt="Screen Shot 2020-05-13 at 12 51 29 PM" src="https://user-images.githubusercontent.com/10120600/81848238-6dad8c80-951a-11ea-9293-57fa02954841.png">

Reply:
Changes have been done, please review